lib/config/options/index.ts

```diff
@@ -1890,7 +1890,7 @@ const options: RenovateOptions[] = [
     description:
       'Text added here will be placed last in the PR body, with a divider separator before it.',
     type: 'string',
-    default: `This PR has been generated by [Renovate Bot](https://github.com/renovatebot/renovate).`,
+    default: `[Renovate](https://github.com/renovatebot/renovate) created this PR.`,
```

Comment:
I'm opening this PR to use better words for the PR footer. I don't like `This PR has been created by ...`, it feels too passive. I like `Renovate created this PR.` better, as it's shorter, and easier to understand.

Our Troubleshooting docs have a reference to the Mend hosted app version of the PR footer though: [^quote]

> The text you're looking for is:
>
> > This PR has been generated by Mend Renovate. View repository job log here.

I can't find the code for that PR footer in the `renovatebot/renovate` repository's code. I guess one of the Mend admins would need to make this change. 😉 

[^quote]: https://docs.renovatebot.com/troubleshooting/#github-hosted-app